enhancements/microshift/microshift-updateability-ostree.md

```diff
@@ -0,0 +1,940 @@
+---
+title: microshift-updateability-ostree
+authors:
+  - "@pmtk"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@pacevedom, MicroShift team"
+  - "@ggiguash, MicroShift team"
+  - "@copejon, MicroShift team"
+  - "@dusk125, etcd team"
```

Comment:
It's because we back up etcd data in unusual way (cp instead of etcdctl snapshot), we already discussed this with @dusk125

Reply:
The EP template asks for:

Include a comment about what domain expertise a reviewer is expected to bring *and what area of the enhancement you expect them to focus on*

Since it sounds like you had the discussion w/ the etcd team directly, you've covered the intent, but in general i'd expect to see something like "etcd team - to focus on the etcd backup/restore implementation plan" or "etcd team - to focus on storage migration approach"  (the intent is to ensure that when the etcd team comes to look at your EP, they can prioritize the parts that are relevant to them)

Reply:
Fixed